src/app/repositories/tasks.py

```diff
@@ -8,4 +8,37 @@
 
 
 class AbstractTaskRepository(ABC):
-    ...
+    @abstractmethod
+    async def get(self, obj_id: Task.id) -> TaskSchema:
+        raise NotImplementedError
+
+    @abstractmethod
+    async def put(self, obj_in: TaskSchema, **kwargs: dict) -> TaskSchema:
+        raise NotImplementedError
+
+    @abstractmethod
+    async def delete(self, db_obj: TaskSchema) -> TaskSchema:
+        raise NotImplementedError
+
+
+class SqlAlchemyTaskRepository(AbstractTaskRepository):
+    def __init__(self, model: Task, session: AsyncSession) -> None:
```

Comment:
модель необязательно передавать, но если так больше нравится ток ок. лучше тогда сделай так:
```suggestion
    def __init__(self, session: AsyncSession, model: Task = Task) -> None:
```

иначе у тебя в функции `create_tasks_service` будет исключение